package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage && npm run snyk",
+    "test": "npm run lint && npm run unit && npm run cover && npm run check-coverage",
```

Comment:
I agree it's not good.

There are multiple major version numbers of minimatch being used throughout the dependencies; versions 0.x.x and 2.x.x. Version 3.0.2 is the latest. Some packages appear like they're going to take a little while to update, https://github.com/mochajs/mocha/issues/2318 but hopefully updates will be made to Instanbul soon https://github.com/gotwarlost/istanbul/issues/638. So adding our own dependency won't solve this (and not all of us are using npm 3 unfortunately).

Do you have any other suggestions?